SPECS/coredns/coredns.spec

```diff
@@ -17,8 +17,11 @@ Source0:        %{name}-%{version}.tar.gz
 #   1. wget https://github.com/coredns/coredns/archive/v%%{version}.tar.gz -O %%{name}-%%{version}.tar.gz
 #   2. tar -xf %%{name}-%%{version}.tar.gz
 #   3. cd %%{name}-%%{version}
-#   4. go mod vendor
-#   5. tar  --sort=name \
+#  >>>> Required for Patch1: CVE-2023-44487.patch
+#   4. patch -p1 CVE-2023-44487.patch
+#  >>>>
+#   5. go mod vendor
+#   6. tar  --sort=name \
```

Comment:
I think the `generate_source_tarball.sh` script is taking care of these steps now. I think we can just remove this whole comment and add the lines you've added here somewhere inside that script.

Also, I think we actually need to update the script to work with our automation.